src/main/java/edu/kis/vh/nursery/DefaultCountingOutRhymer.java

```diff
@@ -2,31 +2,64 @@
 
 public class DefaultCountingOutRhymer {
 
-	private int[] NUMBERS = new int[12];
+	private static final int ERROR_VALUE = -1;
 
-	public int total = -1;
+	private static final int NUMBERS_SIZE = 12;
 
+	private final int[] numbers = new int[NUMBERS_SIZE];
+
+	private int total = -1;
```

Comment:
Still some constants are not extracted. Have you asked for review hints (1.4.w7): tiny.cc/998muz